app/controllers/ontologies_metadata_curator_controller.rb

```diff
@@ -0,0 +1,59 @@
+class OntologiesMetadataCuratorController < ApplicationController
+    layout :determine_layout
+
+
+
+    def result
+        @ontologies_ids = params[:ontology][:ontologyId].drop(1)
+        @metadata_sel = params[:search][:metadata].drop(1)
+        @ontologies = []
+        @submissions = []
+        @ontologies_ids.each do |data|
+            @ontologies << LinkedData::Client::Models::Ontology.all.detect {|e| e.acronym == data}
+            @submissions << LinkedData::Client::Models::OntologySubmission.all.detect {|e| e.ontology.acronym == data}
+        end
+        if @ontologies.empty?
+            @ontologies = LinkedData::Client::Models::Ontology.all.sort_by {|e| e.name}
```

Comment:
don't call `LinkedData::Client::Models::Ontology.all` each time same it inside a variable.

`LinkedData::Client::Models::Ontology.all` will call each time the data base for the same thing